src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -3,192 +3,215 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 using System;
+using System.ComponentModel;
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.SqlServer.Dac;
+using System.Reflection;
+using System.Collections.Generic;
 
 namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
 {
     /// <summary>
-    /// Class to define deployment options. 
+    /// Class to define deployment option default value and the description
+    /// </summary>
+    public class DeploymentOptionProperty<T>
+    {
+        public DeploymentOptionProperty(T value, string description = "", string displayName = "")
+        {
+            this.Value = value;
+            this.Description = description;
+            this.DisplayName = displayName;
+        }
+
+        public T Value { get; set; }
+        public string Description { get; set; } = string.Empty;
+
+        // To Diaply the options in ADS extensions UI in SchemaCompare/SQL-DB-Project/Dacpac extensions
```

Comment:
1. Comments should be XML doc (`///`) format
2. You spelled display incorrectly